Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the manager binary
+FROM golang:1.13 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/ pkg/
+COPY templates/ templates/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:nonroot
```

Comment:
I don't think we can use gcr.io/distroless. I actually switch the mariadb-operator back to something similar like we had before: https://github.com/openstack-k8s-operators/mariadb-operator/blob/master/Dockerfile

The main reason is the scripts we use to create the CSV files depend on a few extra binaries.

Also you'll need the csv-generator here.

Reply:
Updated thanks!